helpers/article.js

```diff
@@ -0,0 +1,106 @@
+import uniqid from 'uniqid';
+import Joi from 'joi';
+import PassportHelper from './passport';
+import db from '../models';
+
+const { Article, User } = db;
+
+/**
+ * @exports ArticleHelper
+ * @class ArticleHelper
+ * @description Article Helper
+ * */
+class ArticleHelper {
+  /**
+     * Create a slug based on title
+     * @param {string} title - a title
+     * @return {string} Returns a slug created
+     * @static
+     */
+  static createSlug(title) {
+    const slug = `${slugify(title, { lower: true })}-${uniqid.process()}`;
+    return slug;
+  }
+
+  /**
+     * Check if it is a Article
+     * @param {object} article - an article
+     * @return {boolean} Returns if true if it is valid else return false
+     * @static
+     */
+  static isValidArticle(article) {
+    const schema = Joi.object().keys({
+      body: Joi.string().required(),
+      title: Joi.string().required(),
+      description: Joi.string().required(),
+      image: Joi.string().required()
+
+    });
+    return Joi.validate(article, schema);
+  }
+
+  /**
+     * update an article
+     * @param {object} article - an article
+     * @return {boolean} Returns if true if it is valid else return false
+     * @static
+     */
+  static isValidUpdatedArticle(article) {
+    const schema = Joi.object().keys({
+      body: Joi.string().min(3),
+      title: Joi.string().min(3),
+      description: Joi.string().min(3),
+      image: Joi.string().min(3)
+
+    });
+    return Joi.validate(article, schema);
+  }
+
+  /**
+     * Return Invalid data message
+     * @param {object} res - an object with response
+     *@param{object} result - an object with all errors
+     *@return {res} Returns a response with error
+     * @static
+     */
+  static invalidDataMessage(res, result) {
+    const errors = [];
+    for (let index = 0; index < result.error.details.length; index += 1) {
+      errors.push(result.error.details[index].message.split('"').join(' '));
+    }
+    return res.status(422).send({ status: 422, Error: errors });
+  }
+
+  /**
+     * Return true if id matches
+     * @param {Integer} authUser - an ID of authenticated user
+     *@param{Integer} Owner - an ID of owner
+     *@return {boolean} Return true if id matches else return false
+     */
+  static checkId(authUser, Owner) {
+    if (authUser === Owner) {
+      return true;
+    }
+
+    return false;
+  }
+
+  /**
+     * Return if the user is owner
+     * @param {object} req - an object
+     * @param {object} res - an object
+     *@return {boolean} Return true if user matche
+     */
+  static async foundArticleAndOwner(req) {
+    const currentSlug = req.params.slug;
+    const isExist = await Article.findOne({ where: { slug: currentSlug } });
+    if (!isExist) {
+      return 'Article Not found';
+    }
+    const authUser = await PassportHelper.findRecord(User, req.user.id);
+    const isOwner = ArticleHelper.checkId(authUser.dataValues.id, isExist.dataValues.UserId);
```

Comment:
```suggestion
    const isOwner = authUser.dataValues.id === isExist.dataValues.UserId;
```
Then delete `checkIn` function